Improve the connector example to handle the file connectors being removed form the classpath (#6709)
strimzi/strimzi-kafka-operator: documentation/assemblies/deploying/assembly-deploy-kafka-connect-with-plugins.adoc

[id='using-kafka-connect-with-plug-ins-{context}']
= Extending Kafka Connect with connector plugins

> Up until Apache Kafka 3.1.0, the Strimzi container images for Kafka Connect include two built-in file connectors for moving file-based data into and out of your Kafka cluster.

Reviewer: I'm not sure we need lines 8-25 now? 
If we want to keep it, do we need the versions?
Maybe this would be sufficient for the intro:

```
Kafka provides example file connectors for moving file-based data into and out of your Kafka cluster. 

.File connectors
[cols="2*",options="header",stripes="none",separator=¦]
|===
¦File Connector
¦Description
m¦FileStreamSourceConnector
¦Transfers data to your Kafka cluster from a file (the source).
m¦FileStreamSinkConnector
¦Transfers data from your Kafka cluster to a file (the sink).

|===

The procedures in this section describe how you can add the example connectors or your own connectors by doing one of the following:
```

> Up until Apache Kafka 3.1.0, the Strimzi container images for Kafka Connect include two built-in file connectors for moving file-based data into and out of your Kafka cluster.

Reviewer: ```suggestion
Up until Apache Kafka 3.1.0, the Strimzi container images for Kafka Connect included two built-in file connectors for moving file-based data into and out of your Kafka cluster.
```

|===

From Apache Kafka 3.1.1 and 3.2.0, these connectors are not included anymore.

> The procedures in this section show how to add your own connector classes to connector images by:

Reviewer: ```suggestion
The procedures in this section describe how you can add the example connectors or your own connectors by doing one of the following:
```

|===

> From Apache Kafka 3.1.1 and 3.2.0, these connectors are not included anymore.

Reviewer: ```suggestion
From Apache Kafka 3.1.1 and 3.2.0, these connectors are no longer included.
```
